### DISCUSSION TITLE
Is Fastlane 3.0 still in progress?

### DISCUSSION THREAD

**Discussion by changusmc** [fastlane/fastlane#17819] [General]
Hi, 

I'm trying to track the process of Fastlane 3.0 and the milestone for it has no tasks and all the tasks in them are closed https://github.com/fastlane/fastlane/milestone/14

Thanks,
David

**Comment by dinsen** [comment]
`Spaceship::ConnectAPI` have been released. But the `Spaceship::TestFlight` has not been removed.
Thats properly why it's not called 3.0.0 yet.

https://docs.fastlane.tools/app-store-connect-api/

**Comment by rogerluan** [comment]
We started some discussions here @changusmc @dinsen 😊  https://github.com/fastlane/fastlane/discussions/20463
